src/main/groovy/com/palantir/gradle/javadist/DistTarTask.groovy

```diff
@@ -57,6 +57,8 @@ class DistTarTask extends Tar {
             }
         }
 
+        new File(project.projectDir, "var/tmp/files").mkdirs()
```

Comment:
sorry i didn't see this earlier.  i strongly recommend against using var/tmp/files.  responded in the thread